lib/cn/wbt_builder.c

```diff
@@ -56,6 +56,7 @@ struct wbb {
     uint  max_inodec;
     uint  max_pgc;
     uint  used_pgc;
+    bool  finalized;
```

Comment:
@gsramdasi, please check on changes to this file.

Reply:
Looks good.